Command/RemoveThumbsCommand.php

```diff
@@ -163,11 +157,12 @@ public function getProvider()
     public function getContext()
     {
         $context = $this->input->getArgument('context');
+
         if (null === $context) {
             $contexts = array_keys($this->getMediaPool()->getContexts());
-            $dialog = $this->getHelperSet()->get('dialog');
-            $contextKey = $dialog->select($this->output, 'Please select the context', $contexts);
-            $context = $contexts[$contextKey];
+
+            $question = new ChoiceQuestion('Please select the context', $contexts);
```

Comment:
This variable seems to be use only one time.

It should not be initialized.

cc @greg0ire